source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -21,6 +21,26 @@ local getListOrdinality(listName) = {
     },
   ],
 };
+local newOrdinal(listName) = {
```

Comment:
Perhaps this needs a different name which makes it clear why it's different from the `getListOrdinality` rule?